news/templates/news/user_news.html

```diff
@@ -1,47 +1,41 @@
 {% load i18n %}
 {% load news %}
 
-{% for item in alerts %}
-<div class="alert alert-{{ item.alert }}">
-  <h4>
-    {{ item.title|safe }}
-    {% if not item|is_published:now %}<small><span class="label label-default">{% trans "Appears" %} {{ item.publish }}</span></small>{% endif %}
-    {% if is_course_staff %}<small><span class="label label-default">{{ item.audience|news_audience }}</span></small>{% endif %}
-  </h4>
-  <div>
-    {{ item.body|safe }}
-  </div>
-</div>
-{% endfor %}
 
 {% if news %}
-<div class="panel panel-default news-panel">
-  <div class="panel-heading">
-    <h4 class="panel-title">{% trans "Course news" %}</h4>
-  </div>
-  <div class="list-group">
-    {% for item in news %}
-    <a class="list-group-item{% if item.collapsed %} collapsed{% endif %}" href="#news{{ item.id }}" data-toggle="collapse" role="button" aria-controls="news{{ item.id }}" aria-expanded="{% if item.collapsed %}false{% else %}true{% endif %}">
-      <h4 class="list-group-item-heading">
-        <span class="caret"></span>
-        {{ item.title|safe }} <small>{{ item.publish }}</small>
-        {% if not item|is_published:now %}<small><span class="label label-default">{% trans "Appears later" %}</span></small>{% endif %}
-        {% if is_course_staff %}<small><span class="label label-default">{{ item.audience|news_audience }}</span></small>{% endif %}
-      </h4>
-    </a>
-    <div class="list-group-item-text collapse{% if not item.collapsed %} in{% endif %}" id="news{{ item.id }}">
-      {{ item.body|safe }}
-    </div>
-    {% if item.begin_more %}
-    <a class="list-group-item" href="#more-news" data-toggle="collapse" role="button" aria-controls="more-news" aria-expanded="false">
-      {% trans "Show more" %}
-    </a>
-    <div class="collapse" id="more-news">
-    {% endif %}
-    {% endfor %}
-    {% if more > 0 and news|length > more %}
-    </div>
-    {% endif %}
-  </div>
-</div>
+	<div class="panel panel-default news-panel">
+		<div class="panel-heading">
+			<h3 class="panel-title">{% trans "Course news" %}</h3>
+		</div>
+		<div class="list-group">
+			{% for item in news %}
+				<div class="{% if item.pin %}pinned-{% endif %}list-group-item">
+					<h4 class="list-group-item-heading">
```

Comment:
I'm on the fence about whether it makes sense for the `h4` element to include both the headline and the date. If you do the thing mentioned on line 18, maybe you could try moving it outside of the heading element?

Reply:
I moved it outside the h4-tag, while trying to keep the css minimum and classes still explanatory